### PR TITLE
Don't obsolete the v2 pickers, renamed them and they're able to be created again, without config changes

### LINKED ISSUE #10423: Preserve Mediapicker v2 so that it's not obsolete
### Which Umbraco version are you using?

8.14.0

### Bug summary

With regards to https://github.com/umbraco/Umbraco-CMS/issues/10397 we will "un-obsolete" MediaPicker v2 to live happily side-by-side with Media Picker v3.

### Specifics

_No response_

### Steps to reproduce

n/a

### Expected result / actual result

_No response_

---
_This item has been added to our backlog [AB#12354](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/12354)_

### PR DESCRIPTION
**Pull request by nul800sebastiaan**
Fixes #10423

To test: 
- Do a fresh install and notice the media picker is there named `Media Picker (legacy)` - if you install the starter kit, this is what pickers are use
- Upgrade an 8.13 site, notice that the picker is still names `Media Picker` without `(legacy)` since we don't do any renaming during upgrades
- In either install, notice that the `legacy` picker can still be created, without config changes

### CODE REVIEW

File: src/Umbraco.Web/PropertyEditors/MediaPickerPropertyEditor.cs [hunk 1]
```diff
@@ -8,17 +8,17 @@ namespace Umbraco.Web.PropertyEditors
 {
     /// <summary>
     /// Represents a media picker property editor.
-    /// Marked as Deprecated as best to use the NEW Media Picker aka MediaPicker3
+    /// Nameed "(legacy)" as it's best to use the NEW Media Picker aka MediaPicker3
```
Comment (bjarnef) [inline src/Umbraco.Web/PropertyEditors/MediaPickerPropertyEditor.cs:11]: @nul800sebastiaan a minor typo in "Named" 😁

Comment (warrenbuckley) [reply src/Umbraco.Web/PropertyEditors/MediaPickerPropertyEditor.cs:11]:
```suggestion
    /// Named "(legacy)" as it's best to use the NEW Media Picker aka MediaPicker3
```

**Comment by warrenbuckley** [comment]
Code changes fine, doing a test :)

**Review by warrenbuckley — APPROVED** [review]
Works fine Seb

**Pull request merged**